content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,293 @@
+    <br />
+
+    ```bash
+    docker run -it --rm \
+    --net=host \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/opt/spectrocloud \
+    gcr.io/spectro-images-public/release/spectro-installer:1.0.12 \
+    -o true
+    ```
+
+6. When prompted, enter the pairing code and information listed in each of the following tables. The installer will generate the gateway configuration file. 
+    <br />
+
+
+#### Palette Parameters
+
+|**Parameter**       | **Description**|
+|:-----------------------------|---------------|
+|**Install Type**| Choose **Private Cloud Gateway** or **Self Hosted Enterprise Cluster**. <br />You can change your selection with the up or down keys.|
+|**Cloud Type**| Choose MAAS.|
+|**Name** | Enter a custom name for the PCG.|
+|**Endpoint** |Enter the Palette endpoint. Example: https://customername.console.spectrocloud.com. |
+|**Username** |Enter your Palette username. This is your sign-in email address. Example: user1@company.com. |
+|**Password** |Enter your Palette Password. This is your sign-in password.|
+|**Private Cloud Gateway** |Enter the PCG pairing code you copied from the instructions page in step **4**. |
+
+<br />
+
+#### Environment Configuration
+
+
+|**Parameter**| **Description**|
+|:-------------|----------------|
+|**HTTPS Proxy (--https_proxy)**| Leave this blank unless you are using an HTTPS Proxy. This setting will be propagated to all the nodes launched in the proxy network. Example: ``https://USERNAME:PASSWORD@PROXYIP:PROXYPORT``.|
+| **HTTP Proxy(--http_proxy)**| Leave this blank unless you are using an HTTP Proxy. This setting will be propagated to all the nodes launched in the proxy network. Example: ``http://USERNAME:PASSWORD@PROXYIP:PROXYPORT``.|
+| **No Proxy(--no_proxy)**| The default is blank. You can add a comma-separated list of local network classless inter-domain routing (CIDR) addresses, hostnames, and domain names that should be excluded from being a proxy. This setting will be propagated to all the nodes to bypass the proxy server. Example: ``maas.company.com,10.10.0.0/16``.|
+| **Pod CIDR (--pod_cidr)**|Enter the CIDR pool that will be used to assign IP addresses to pods in the cluster. The pod IP addresses should be unique and should not overlap with any virtual machine IPs in the environment.|
+| **Service IP Range (--svc_ip_range)**|Enter the IP address range that will be used to assign IP addresses to services in Kubernetes clusters. The service IP addresses should be unique and not overlap with any virtual machine IPs in the environment.|
+
+<br />
+
+
+#### MAAS Account Information
+
+<WarningBox>
+
+When specifying the MAAS endpoint, do not add a trailing `/` as it can result in DNS resolution errors.
+
+</WarningBox>
+
+|**Parameter**| **Description**|
+|-------------|----------------|
+| **API Endpoint** |Enter the MAAS API endpoint without a trailing /. This can be a domain or IP address. Example: ``http://10.11.12.13:5240/MAAS.``|
```

Comment:
```suggestion
| **API Endpoint** |Enter the MAAS API endpoint without a trailing `/`. This can be a domain or IP address. Example: ``http://10.11.12.13:5240/MAAS.``|
```